notifier/notifier.go

```diff
@@ -160,6 +160,12 @@ func (notifier *StandardNotifier) runSender(sender moira.Sender, ch chan Notific
 				notifier.logger.Errorf(buildErr)
 			}
 		}
+
+		desc, err := pkg.Trigger.PopulateDescription(pkg.Events)
+		if err == nil {
+			pkg.Trigger.Desc = desc
```

Comment:
Maybe log a warning if `err` is not `nil`? It's wrong to silently ignore malformed descriptions.